### DISCUSSION TITLE
練習markdown語法表情貼圖

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#152] [General]
我發現在Discord中，各式圖案都有附markdown語法。這樣在Github上要使用圖案就非常方便。:sunny:=:sheep:=:meat_on_bone:=:100:喜歡:hamster:、:turtle:、:lizard:與:sauropod:。

**Comment by enoch-lightning** [comment]
🐶 🐕 🦮 🦮 🐩 🐕‍🦺 🐱 🐈 😸 🥇 👍

**Comment by Jie2222** [comment]
[圖案列表~](https://www.webfx.com/tools/emoji-cheat-sheet/)
👍
